src/Apps/About/AboutApp.tsx

```diff
@@ -107,8 +107,8 @@ const Section: React.FC<React.PropsWithChildren<SectionProps & BoxProps>> = ({
   const image = resized(imageUrl, { width: 640 })
 
   return (
-    <RouterLink to={href} textDecoration="none" display="block">
-      <Box {...rest}>
+    <Box {...rest}>
+      <RouterLink to={href} textDecoration="none" display="block">
```

Comment:
Not sure exactly why this is suddenly failing our smoke test, but it also fails in main? Odd. In any case, this is not valid markup.